src/Hero6.DesktopGL/Hero6.DesktopGL.csproj

```diff
@@ -15,6 +15,7 @@
     <MonoGameContentBuilderExe Condition=" '$(OS)' != 'Windows_NT' And Exists ('/opt/monogame-pipeline/MGCB.exe') ">/opt/monogame-pipeline/MGCB.exe</MonoGameContentBuilderExe>
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <TargetFrameworkProfile />
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
If you're willing to accept this PR, the drop down has an option to squash and merge.